driver/meminfo/meminfo.h

```diff
@@ -71,13 +71,15 @@ class MemInfo final : public task::Task {
             .scale = 1.2
         };
 
-        pipe = pipeline::Acquisition(
-            ctx,
-            writer_cfg,
-            std::move(source),
-            breaker_config
-        );
-        pipe.start();
+
+        // TODO: change this so that you are not using the acquisition pipeline copy constructor (follow ni pattern w/ configure)
+        // pipe = pipeline::Acquisition(
```

Comment:
Why is this commented out. Did we just turn off the meminfo task?

Reply:
i did temporary cus i needed to make some changes. ive just made those changes and included it in main again.